Misc/NEWS.d/next/Library/2022-09-13-11-54-12.gh-issue-NA.AgVTrC.rst

```diff
@@ -0,0 +1 @@
+Updated ast.py to use f-string syntax.
```

Comment:
This file should be deleted. We only add news for existing github issues.

Reply:
Will do.